peer_mentoring/groups/views.py

```diff
@@ -52,6 +53,9 @@ def groups_moderated(request):
 def group_detail(request, group_id):
     group = get_object_or_404(Group, pk=group_id)
     posts = Post.objects.filter(group=group_id)
+    comment_count = Post.objects.filter(group=group_id).annotate(
```

Comment:
this is correct, you can now use comment_count as attribute on the post, so that probably still needs fixing in the template as I don't see the count when adding 2 comments on a post ...

```
(Pdb) comment_count
<QuerySet [<Post: post1>]>
(Pdb) comment_count[0]
<Post: post1>
(Pdb) comment_count[0].comment_count
2
```

Reply:
Please merge this in, did a quick PR with fix here: https://github.com/JAWD-001/peer_mentoring/pull/70

HTH
Bob